README.md

```diff
@@ -1,6 +1,6 @@
 # UX Writer, Technical Writer, and Instructional Designer GitHub Site
 
-If you're familiar with Git/GitHub then head to the Wiki portion of this GitHub site. If you are unfamiliar with Git/GitHub,  this is your starting point. Most of the relevant items are contained within the Wiki portion of this site, but I have also included information here. 
+If you're familiar with Git/GitHub then head to the [Wiki portion of this GitHub site](https://github.com/techuxwriter/UX-Tech-Writer-IsD-GitHub/wiki). If you are unfamiliar with Git/GitHub,  this is your starting point. Most of the relevant items are contained within the Wiki portion of this site, but I have also included information here. 
```

Comment:
Line 3 - Suggested Comment: @techuxwriter - I think it would be helpful to include a direct link to the Wiki since that is the place where most relevant content is contained.

Reply:
@kamahoney1965 - Thanks for the suggestion, I agree. :)